api/kessel/relations/v1beta1/check.proto

```diff
@@ -1,22 +1,22 @@
 syntax = "proto3";
 
-package kessel.relations.v0;
+package kessel.relations.v1beta1;
 
 import "google/api/annotations.proto";
-import "relations/v0/common.proto";
+import "kessel/relations/v1beta1/common.proto";
 import "validate/validate.proto";
 
 
-option go_package = "github.com/project-kessel/relations-api/api/relations/v0";
+option go_package = "github.com/project-kessel/relations-api/api/kessel/relations/v1beta1";
 option java_multiple_files = true;
-option java_package = "org.project_kessel.api.relations.v0";
+option java_package = "org.project_kessel.api.relations.v1beta1";
```

Comment:
This will probably end up a breaking change on the Java clients (if I understand their generation correctly). Not a huge deal, just something we'll need to keep in mind when we update the client lib and make sure we give the Java teams a heads up about.